Tasks/PublishTestResults/task.loc.json

```diff
@@ -15,10 +15,10 @@
   "version": {
     "Major": 1,
     "Minor": 0,
-    "Patch": 5
+    "Patch": 6
   },
   "demands": [],
-  "minimumAgentVersion": "1.83.0",
+  "minimumAgentVersion": "1.88.0",
```

Comment:
I suggest not to bump up the agent version required. if users wants to use xunit we shuld just print good error message and then user can update. for other runners like vstest and nunit and junit we shuld ask users to update version